lib/apr/payments.ex

```diff
@@ -3,17 +3,44 @@ defmodule Apr.Payments do
 
   alias Stripe.PaymentIntent
 
-  def liability_shift_happened(nil), do: false
+  def payment_info(external_id, "payment_intent") do
```

Comment:
Updated our logic to get all Stripe's related info from a payment intent in one method and encapsulate Stripe's specific logic and structure in this module, so from caller's perspective they always get same response back from here regardless of stripe changes.